workflows/devnet-build/Jenkinsfile

```diff
@@ -77,8 +84,8 @@ pipeline{
                             sha256sum ${outputFile} > ${outputFile}.SHA256
                             cd ..
                         fi
-                        s3cmd put --acl-public release/${outputFile} s3:///lisk-releases/lisk-core/${outputFile}
-                        s3cmd put --acl-public release/${outputFile}.SHA256 s3:///lisk-releases/lisk-core/${outputFile}.SHA256
+                        s3cmd put --acl-public release/${outputFile} s3:///lisk-releases/lisk-core/${params.TAG}/${outputFile}
```

Comment:
Did we not decide to put all >=1.1 builds directly under `/lisk-core/` since they are not network specific any more?

Reply:
That would be awesome. For some reason, the Jenkins builds weren't uploading there

Reply:
That said, the builds are still made with "dev" or "test" or "alpha", so we might want a way to differentiate between them?